components/CreateChain/QuestsForm.tsx

```diff
@@ -249,7 +251,7 @@ export const QuestsForm: React.FC<{
           </SubmitButton>
         )}
         <SubmitButton
-          isDisabled={isPremium && !isApproved}
+          isDisabled={(isPremium && !isApproved) || isSubmitting}
```

Comment:
Should we add `isLoading={isSubmitting}` too ?

Reply:
yup, will add!

Reply:
done 👌